programs/locked-voter/src/state.rs

```diff
@@ -77,12 +92,19 @@ pub struct Escrow {
     pub is_max_lock: bool,
     /// total amount of partial unstaking amount
     pub partial_unstaking_amount: u64,
-    /// padding for further use
-    pub padding: u64,
+
+    // Before recovery approval, freezing is to stop leaked keys from withdrawing.
+    // After recovery approval, freezing is to
+    /// Frozen
+    pub freeze_timestamp: i64,
+    /// New ta to recover into
+    pub recovery_key: Pubkey,
     /// buffer for further use
-    pub buffers: [u128; 9],
+    pub buffers: [u8; 31],
```

Comment:
hmm, why buffers after is [u8; 31]? 
Pubkey = [u8; 32] =[u128; 2]
So buffers after should be [u128; 7]

Reply:
u128 -> 7 bits 
7*9 -> 63
63 - 32 = 31

Reply:
wait, i'm confused. 
u128 -> 128/8 => 16 bytes 
I see